main_test.go

```diff
@@ -95,3 +95,60 @@ func TestCopyFromCommandLineArgument(t *testing.T) {
 		t.Errorf("Expected %s but got %s", content, copied)
 	}
 }
+
+func TestParseContentDirectText(t *testing.T) {
+	directText := "Direct text input"
+	contentStr, err := parseContent(&directText, []string{})
+	if err != nil {
+		t.Errorf("Expected no error, got %v", err)
```

Comment:
```suggestion
		t.Fatalf("Expected no error, got %v", err)
```

There is usually no need to continue when you get an error